run-docker.sh

```diff
@@ -25,7 +25,7 @@
 # you use; `activate` is for bash but there are different suffixed files for
 # other shell types.
 CONTAINER_NAME=${SEARCHFOX_DOCKER_CONTAINER_NAME:-searchfox}
-IMAGE_NAME=${SEARCHFOX_DOCKER_IMAGE_NAME:-searchfox}
+IMAGE_NAME=localhost/${SEARCHFOX_DOCKER_IMAGE_NAME:-searchfox}
```

Comment:
It would be good to add a comment here explaining what's going on or linking to docs somewhere that explain what's going on.

Restating:
- I understand maybe podman adds this prefix to images it builds which potentially makes it necessary?
- This seems to be harmless for my non-podman use-case based on local testing, presumably because I guess docker understands the pre-slash piece to potentially be a reference to a remote repository (and this explicitly is not remote)

Reply:
> linking to docs somewhere that explain what's going on

I've added that now: linking to both docker and podman docs which describe that syntax and why it avoids querying a remote.